### ISSUE TITLE
Is there a way to refresh a SELECT element after setting selectedIndex?

### ISSUE THREAD

**Issue by divadr** [blackberry/bbUI.js#52]
First, thanks very much for creating bbUI, it's fast, looks good, and is fun to work with!  

I have a very simple app that I'm using to get my teeth wet with Bb HTML5 development.  On an HTML screen there are two <select> or drop down boxes, and a single button on the page.  The two <select> elements are exactly the same except for their ID, and NAME properties.  The purpose of the button is to simply change the selectedIndex of each select element for the selectedindex value of the other select element, or invert their selectedindex values.  

Here are the select elements:

```
<select name="SelectA" id="SelectA" onChange="doSomething();" onKeypress="if(event.keyCode == 13) {doSomething();};" onClick="doSomething();">
    <option value="0">0</option>
    <option value="1">1</option>
    <option value="2" selected="selected">2</option>
    <option value="3">3</option>
    <option value="4">4</option>
    <option value="5">5</option>
    <option value="6">6</option>
    <option value="7">7</option>
    <option value="8">8</option>
    <option value="9">9</option>
</select>

<select name="SelectB" id="SelectB" onChange="doSomething();" onKeypress="if(event.keyCode == 13) {doSomething();};" onClick="doSomething();">
    <option value="0">0</option>
    <option value="1">1</option>
    <option value="2">2</option>
    <option value="3">3</option>
    <option value="4">4</option>
    <option value="5">5</option>
    <option value="6" selected="selected">6</option>
    <option value="7">7</option>
    <option value="8">8</option>
    <option value="9">9</option>
</select>
```

Here's the button element which calls the fnInvert function with the onClick event:

```
<div data-bb-type="button" id="btninvert" onclick="fnInvert(document.getElementById('SelectA'), document.getElementById('SelectB'));">invert</div>
```

Here's the fnInvert function which swaps the selectedIndex (I tried to refresh the select by setting the selectedIndex to -1, and also tried setting the specific option from the options collection to true):

```
function fnInvert (From, To) {
    var FromValue = From.selectedIndex;
    var ToValue = To.selectedIndex;

    // From.options[ToValue].selected = true;
    From.selectedIndex = -1;
    To.selectedIndex = -1;
    From.selectedIndex = ToValue;
    To.selectedIndex = FromValue;
};
```

Basically, the fnInvert function works.  When I click the button the selectedIndex values are swapped between the two select elements.  However the change is not noted visibly.  The fact that the selectedIndex has changed is not discernible as the originally selected, or visible option does not change.  However, when I click the select element, the option menu appears and the option that is checked is the option that was changed by the fnInvert function.  

Is there a way to "refresh" the select element?  I know that jQuery offers a refresh method for the select element, but I haven't looked into why that's the case.
